src/cosmosdb-continuous-backup/azext_cosmosdb_pitr/_params.py

```diff
@@ -0,0 +1,112 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long, too-many-statements
+
+from enum import Enum
+
+from azure.cli.core.commands.parameters import (
+    get_resource_name_completion_list, name_type, get_enum_type, get_three_state_flag, get_datetime_type, get_location_type)
+
+from azext_cosmosdb_pitr._validators import (
+    validate_capabilities, validate_virtual_network_rules, validate_ip_range_filter)
+
+from azext_cosmosdb_pitr.actions import (
+    CreateLocation, CreateDatabaseRestoreResource)
+
+from azext_cosmosdb_pitr.vendored_sdks.azure_mgmt_cosmosdb.models import DefaultConsistencyLevel, DatabaseAccountKind, ServerVersion
+
+
+class BackupPolicyTypes(str, Enum):
+    periodic = "Periodic"
+    continuous = "Continuous"
+
+
+def load_arguments(self, _):
+
+    from azure.cli.core.commands.parameters import tags_type
+
+    with self.argument_context('cosmosdb') as c:
+        c.argument('account_name', arg_type=name_type, help='Name of the Cosmos DB database account', completer=get_resource_name_completion_list('Microsoft.DocumentDb/databaseAccounts'), id_part='name')
+        c.argument('database_id', options_list=['--db-name', '-d'], help='Database Name')
+
+    with self.argument_context('cosmosdb create') as c:
+        c.argument('account_name', completer=None)
+        c.argument('key_uri', help="The URI of the key vault", is_preview=True)
+        c.argument('enable_free_tier', arg_type=get_three_state_flag(), help="If enabled the account is free-tier.", is_preview=True)
```

Comment:
would `false` take effect? If not, I will recommend to use `store_true`

Reply:
Actually I copied this code from the source repo (azure-cli) and just added few parameters for `az cosmosdb create` and `az cosmosdb update` commands. 
I did not want to change the behavior of the existing parameters for these commands.  
Do you think it would be better to use `store_true` here but not in source repo?

Reply:
I see. To prevent unexpected behaviors to customer, you could keep it.